main.go

```diff
@@ -31,6 +31,7 @@ func main() {
 	flag.StringVar(&flags.formatter, "fmt", "", "go pretty-printer: gofmt, goimports or noop (default gofmt)")
 	flag.BoolVar(&flags.stubImpl, "stub", false,
 		"return zero values when no mock implementation is provided, do not panic")
+	printVersion := flag.Bool("version", false, "version for moq")
```

Comment:
Should it be `-version` or `-v`?

Reply:
Unfortunately, there is not really a standard on how the flag is named to print the version information of a tool. I have seen `-v`, `-V`, `--version`, `-version` and `version` (for tools with a command style arguments API). Therefore we can choose whatever we like.
Currently we do not have any single letter flag for `moq` so I suggest to follow the current pattern of *named* flags and therefore I would keep the longer form of `-version`.

Reply:
OK, pretty compelling arguments.